tests/baselines/reference/showConfig/Shows tsconfig for single option/locale/tsconfig.json

```diff
@@ -1,5 +1,3 @@
 {
-    "compilerOptions": {
-        "locale": "someString"
-    }
+    "compilerOptions": {}
```

Comment:
Interesting that it was shown before and not now..

Reply:
Yeah, locale _used_ to be an `Advanced Option` which meant it would have passed that previous `if` check - but the flag only works on the CLI (and so it's not in the tsconfig reference for example), so I felt moving it into command line options was probably the right call

```diff
            name: "locale",
             type: "string",
-            category: Diagnostics.Advanced_Options,
+            category: Diagnostics.Command_line_Options,
```

Reply:
It is not command line only option... it can be present in tsconfig ?

Reply:
Just confirmed to make sure, tsc ignores locale in a tsconfig

```
❯ cat tsconfig.json
{
  "files": [
    "broke.ts"
  ],
  "compilerOptions": {
    "locale": "cs",
  }
}

❯ pnpm tsc
broke.ts:2:7 - error TS1134: Variable declaration expected.

2 const 123 = 123
        ~~~


Found 1 error.


❯ pnpm tsc --locale cs
broke.ts:2:7 - error TS1134: Očekává se deklarace proměnné.

2 const 123 = 123
        ~~~


Našla se 1 chyba.
```

and

```
❯ cat tsconfig.json
{
  "files": [
    "broke.ts"
  ],
  "locale": "cs",
}

❯ pnpm tsc
broke.ts:2:7 - error TS1134: Variable declaration expected.

2 const 123 = 123
        ~~~


Found 1 error.
```

It can be considered a bug, but the switch at least makes sense for how TS acts today